draft-ietf-quic-recovery.md

```diff
@@ -1101,22 +1101,24 @@ OnAckReceived(ack, pn_space):
 
 
 UpdateRtt(latest_rtt, ack_delay):
```

Comment:
Why is `latest_rtt` passed in as a parameter? It's already a global. Does this parameter differ from the global?

Reply:
latest_rtt probably shouldn't be global.

Reply:
It's used in `DetectLostPackets`, so it needs to be global.
To avoid confusion with scoping rules, the function parameter should have a different name than the global variable though.